src/test/java/org/broadinstitute/hellbender/utils/fasta/CachingIndexedFastaSequenceFileUnitTest.java

```diff
@@ -179,45 +176,48 @@ private int testCases(final IndexedFastaSequenceFile original,
         }
     }
 
-    private String fetchBaseString(final IndexedFastaSequenceFile reader, final String contig, final int start, final int stop) {
-        if ( start == -1 )
+    private String fetchBaseString(final ReferenceSequenceFile reader, final String contig, final int start, final int stop) {
+        if ( start == -1 ) {
             return new String(reader.getSequence(contig).getBases());
-        else
+        } else {
             return new String(reader.getSubsequenceAt(contig, start, stop).getBases());
+        }
     }
 
     @Test
-    public void testIupacChanges() throws FileNotFoundException, InterruptedException {
+    public void testIupacChanges() throws IOException {
         final String testFasta = publicTestDir + "iupacFASTA.fasta";
-        final CachingIndexedFastaSequenceFile iupacPreserving = new CachingIndexedFastaSequenceFile(IOUtils.getPath(testFasta), CachingIndexedFastaSequenceFile.DEFAULT_CACHE_SIZE, false, true);
-        final CachingIndexedFastaSequenceFile makeNs = new CachingIndexedFastaSequenceFile(IOUtils.getPath(testFasta));
+        try(final CachingIndexedFastaSequenceFile iupacPreserving = new CachingIndexedFastaSequenceFile(IOUtils.getPath(testFasta), CachingIndexedFastaSequenceFile.DEFAULT_CACHE_SIZE, false, true);
+            final CachingIndexedFastaSequenceFile makeNs = new CachingIndexedFastaSequenceFile(IOUtils.getPath(testFasta))) {
```

Comment:
Pass in an explicit `false` for `preserveIUPAC` in the constructor call for `makeNs` here instead of relying on the defaults.

Reply:
done